wisp/wisp.php

```diff
@@ -284,8 +284,9 @@ function wisp_CreateAccount(array $params) {
             if($userResult['meta']['pagination']['total'] === 0) {
                 $userResult = wisp_API($params, 'users', [
                     'email' => $params['clientsdetails']['email'],
-                    'first_name' => $params['clientsdetails']['firstname'],
-                    'last_name' => $params['clientsdetails']['lastname'],
+                    //set first_name and last_name if empty
+                    'first_name' => empty($params['clientsdetails']['firstname']) ? 'Unkown' : $params['clientsdetails']['firstname'],
```

Comment:
Spelling of **Unknown** is wrong here, should be changed from 'Unkown' to 'Unknown'.